functions/src/etl/SurveyManager.ts

```diff
@@ -0,0 +1,122 @@
+import * as functions from 'firebase-functions';
```

Comment:
As long as the surveys are manually created from the admin tool, I would move this file to the `admin/` directory that I created in my PR. What do you think?

Reply:
👍 will do